Plugins/HockeyAppUnityAndroid/HockeyAppUnity-Scripts/HockeyAppAndroid.cs

```diff
@@ -8,6 +8,7 @@
 using System;
 using System.IO;
 using System.Runtime.InteropServices;
+using UnityEngine.Networking;
```

Comment:
Unfortunately, it will not work as is with Unity 5.0 (this is the minimum supported version for this SDK, see [requirements](https://github.com/bitstadium/HockeySDK-Unity-Android/blob/master/README.md#requirements)).

Also, based on [doc](https://docs.unity3d.com/560/Documentation/ScriptReference/Networking.UnityWebRequest.html), it looks like `isNetworkError` was just `isError` in 5.x

So, let's keep backward compatibility and use the new implementation only for 2017.1 or newer. This can be done by wrapping new code in [`UNITY_2017_1_OR_NEWER` block](https://docs.unity3d.com/Manual/PlatformDependentCompilation.html).

Reply:
I've added the `UNITY_2017_1_OR_NEWER` directive, tested in `5.4.0f3`, please review.